### PR TITLE
Release note blog 1.3

### LINKED ISSUE #4790: Knative 1.3 release notes blog post
## Tasks

1. After each component has released, consolidate all release notes into a blog post similar to the [Knative 1.2 blog post](https://knative.dev/blog/releases/announcing-knative-v1-2-release/). Use the [template](https://github.com/knative/docs/blob/main/blog/docs/releases/_template-announcing-knative-v1-X-release.txt) to format the blog post correctly.
2. Add the new blog page to `blog/config/nav.yml`
3. Update the "featured posts" section of `blog/docs/index.md` with the new release. 

## Resources

Add the following release notes to the post:
- [x] https://github.com/knative/serving/releases/tag/knative-v1.3.0
- [x] https://github.com/knative/eventing/releases/tag/knative-v1.3.0
- [x] https://github.com/knative-sandbox/eventing-kafka-broker/releases/tag/knative-v1.3.0
- [x] https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/knative-v1.3.0
- [x] https://github.com/knative/client/blob/main/CHANGELOG.adoc#v130-2022-03-08
- [x] https://github.com/knative/operator/releases/tag/knative-v1.3.0

### PR DESCRIPTION
**Pull request by snneji**
Adds the blog post for 1.3 release notes. I also made a some changes to the heading levels in the template.

I made a guess at which items from the release notes seemed like highlights because it seems like this release mainly consists of various small improvements and bug fixes.

Fixes #4790

### CODE REVIEW

**Comment by snneji** [comment]
/hold

waiting for Knative components to release and post release notes

File: blog/docs/releases/announcing-knative-v1-3-release.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+description: "Knative v1.3 release announcement"
+type: "blog"
+---
+
+## Announcing Knative v1.3 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in the documentation
+[Installing Knative](https://knative.dev/docs/install/) for the respective component.
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-3-release.md:16]:
```suggestion
[Installing Knative](https://knative.dev/docs/install/) for the respective components.
```

(the plural sounds better to my ear, but not 100% that it's grammatically correct...)

File: blog/docs/releases/announcing-knative-v1-3-release.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+Now if the user set the configuration of the RabbitMQ Source Exchange and Queue to be durable, the messages are also durable. ([#560](https://github.com/knative-sandbox/eventing-rabbitmq/pull/560))
+
+
+## Client v1.3
+
+<!-- Original notes are here: https://github.com/knative/client/blob/main/CHANGELOG.adoc#v130-2022-03-08 -->
+
+### 💫 New Features & Changes
+
+- Added Knative Eventtype support. ([#1598](https://github.com/knative/client/pull/1598))
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-3-release.md:136]:
```suggestion
- Added Knative eventtype support. ([#1598](https://github.com/knative/client/pull/1598))
```

(also not 100% on the proper convention here, but looking at the PR docs/comments, seems to be lowercase most of the time)

**Review by psschwei — COMMENTED** [review]
Couple of minor grammar questions (which I may be wrong about), but otherwise LGTM

**Comment by psschwei** [comment]
Also, the post needs to be added to `blog/config/nav.yml`

File: blog/docs/releases/announcing-knative-v1-3-release.md (hunk 1 above)
Comment (snneji) [reply blog/docs/releases/announcing-knative-v1-3-release.md:16]: I think this is referring to installing individual components, but the wording could be clearer. I'll update this.

**Comment by snneji** [comment]
@psschwei I've added the blog post to the `nav.yml` and also updated the index page with this release as a highlighted post.

**Review by psschwei — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**